SEED-platform/seed
# 2374 feat/handle missing raw headers

#### Any background context you want to provide?
Previously, when importing csvs:
- empty header fields were just kept empty as the raw column name
- if there were multiple empty header fields, the last one would essentially 'overwrite' the other empty fields.

#### What's this PR do?
When importing CSVs (not spreadsheets) if there are empty headers, SEED will generate headers for them. Then on the mapping page users are given notice of this with the option to delete the file.

#### How should this be manually tested?
Run the migration and refresh browser cache.
Before each of the following instructions, upload `portfolio-manager-sample-missing-headers.csv`, once on the mapping page you should see a modal warning you that it auto-generated some headers.  

Test 1: click "Continue" on the warning modal, you should see the generated headers as `SEED Generated Header #`. Finish mapping. If you go back to the mapping page you should _not_ see this modal warning.

Test 2: click "Delete File" on the warning modal, you should be redirected to the dataset page and the imported file should be deleted.

#### What are the relevant tickets?
#2374
